example/js/diplomat-runtime.mjs

```diff
@@ -0,0 +1,16 @@
+function readString(wasm, ptr, len) {
```

Comment:
issue (nb): shouldn't be in example/

Reply:
perhaps you put it in the js-tool crate and use `include_str!()` to dump it to a file